java-frontend/src/main/java/org/sonar/java/se/checks/UnclosedResourcesCheck.java

```diff
@@ -116,34 +137,39 @@ private static boolean needsClosing(Type type) {
         return false;
       }
     }
+    for (String excludedType : excludedTypesList) {
+      if (type.is(excludedType)) {
+        return false;
+      }
+    }
     return isCloseable(type);
   }
 
-  private static boolean isCloseable(Type type) {
+  private boolean isCloseable(Type type) {
```

Comment:
I'm not sure you need to remove all of these `static` modifiers. Some of them are probably still valid (this one notably)